packages/auth0/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "homepage": "https://github.com/thefrontside/simulacrum#readme",
   "dependencies": {
+    "@effection/atom": "^2.0.0-beta.3",
```

Comment:
Should this be a dependency, or a dev dependency? Are we importing anything from `@effection/atom`?

Reply:
actually, it was only importing the `Slice<T>` type so I've updated the imports to:

```ts
import type { Slice } from '@effection/atom';
```

And moved it into `devDependencies`